tests/test_data.py

```diff
@@ -1,8 +1,10 @@
 import pytest
 import numpy as np
+import cupy as cp
```

Comment:
Need to be added somehow to the test requirements and this step should be decoratred with a `skipif(device)`